content/en/blog/_posts/2023-10-24-kubernetes-1.28-release-interview.md

```diff
@@ -0,0 +1,223 @@
+Right, so for situations where you have a hardware failure or a broken OS, we have added additional support for a better graceful shutdown.
+
+**If someone trips over the power cord at your LAN party and your cluster goes offline as a result?**<br>
+Right, exactly. More availability! That's always good. 
+
+**And if it's not someone tripping over your power cord, it's probably DNS that broke your cluster. What's changed in terms of DNS configuration?**<br>
+Oh, we introduced [a new feature gate to allow more DNS search path](https://github.com/kubernetes/enhancements/issues/2595).
+
+**Is that all there is to it?**<br>
+That's pretty much it. [LAUGHING] Yeah, you can have more and longer DNS search path.
+
+**It can never be long enough. Just search everything! If .com doesn't work, try .net and try .io after that.**<br>
+Surely.
+
+**Those are a few of the big features that are moving to stable. Obviously, over the course of the last few releases, features come in, moving from Alpha to Beta and so on. New features coming in today might not be available to people for a while. As you mentioned, there are feature gates that you can enable to allow people to have access to these. What are some of the newest features that have been introduced that are in Alpha, that are particularly interesting to you personally?**<br>
+I have two. The first one is [`kubectl delete --interactive`](https://github.com/kubernetes/enhancements/issues/3895). I'm always nervous when I delete something, you know, it's going to be a typo or it's going to be on the wrong tab. So we have an `--interactive` flag for that now.
+
+**So you can get feedback on what you're about to delete before you do it?**<br>
+Right; confirmation is good!
+
+**You mentioned two there, what was the second one?**<br>
+Right; this one is close to my heart. It is a SIG Release KEP, [publishing on community infrastructure](https://github.com/kubernetes/enhancements/issues/1731). I'm not sure if you know, but as part of my branch management associate role in 1.27, I had the opportunity to cut a few releases. It takes up to 12 hours sometimes. And now, we are hoping that the process only includes release managers, so we don't have to call up the folks at Google and, you know, lengthen that process anymore.
+
+**Is 12 hours the expected length for software of this size, or is there work in place to try and bring that down?**<br>
+There's so much work in place to bring that down. I think 12 hours is on the shorter end of it. Unfortunately, we have had a situation where we have to, you know, switch the release manager because it's just so late at night for them. 
+
+**They've fallen asleep halfway through?**<br>
+Exactly, yeah. 6 to 12 hours, I think, is our status quo. 
+
+**The theme for this release is "[Planternetes](/blog/2023-08-15-kubernetes-1.28-blog/kubernetes-1.28.png)". That's going to need some explanation, I feel.**<br>
```

Comment:
The "Planternetes" link is currently leading to a 404 error ([blog preview.](https://deploy-preview-43599--kubernetes-io-main-staging.netlify.app/blog/2023/10/24/plants-process-and-parties-the-kubernetes-1.28-release-interview/)). We should update it to point to the correct location, which is `/images/blog/2023-08-15-kubernetes-1.28-blog/kubernetes-1.28.png`
However, instead of directly linking to the image asset, we could also consider linking to the release blog section, where the release theme logo is displayed (refer to the suggestion below).
```suggestion
**The theme for this release is "[Planternetes](/blog/2023/08/15/kubernetes-v1-28-release/#release-theme-and-logo)". That's going to need some explanation, I feel.**<br>
```

Reply:
Thanks!  I accidentally lost an `images/` when I changed to a relative URL. Thats a good suggestion too.